src/lib.rs

```diff
@@ -90,11 +90,11 @@ The `Deref` implementation uses a hidden static variable that is guarded by an a
 
 # Cargo features
 
-This crate provides two cargo features:
+This crate provides one cargo feature:
 
 - `spin_no_std`: This allows using this crate in a no-std environment, by depending on the standalone `spin` crate.
 
-Both features depend on unstable language features, which means
+This feature depends on unstable language features, which means
```

Comment:
Ah actually this isn't accurate anymore. The `spin_no_std` feature works on the stable Rust channel now. Would you like to just remove this sentence altogether?